### ISSUE TITLE
Cannot sync after last update (2.14)

### ISSUE THREAD

**Issue by zellerda** [aluxnimm/outlookcaldavsynchronizer#186]
````
System.MissingMethodException: Method not found: 'System.String Thought.vCards.vCardDeliveryAddress.get_ExtendedAddress()'.
   at CalDavSynchronizer.Implementation.Contacts.ContactEntityMapper.MapPostalAdresses2To1(vCard source, ContactItem target)
   at CalDavSynchronizer.Implementation.Contacts.ContactEntityMapper.<Map2To1>d__9.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GenSync.Synchronization.States.CreateInA`7.<InitializeEntity>d__13.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GenSync.Synchronization.States.CreateInA`7.JobWrapper.<InitializeEntity>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at CalDavSynchronizer.Implementation.Contacts.OutlookContactRepository`1.<Create>d__18.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at GenSync.EntityRepositories.BatchEntityRepositoryAdapter`4.<PerformOperations>d__2.MoveNext()
````

**Comment by aluxnimm** [comment]
Can you try to manually uninstall and install the latest version, must be some packaging conflict which with the update.

**Comment by zellerda** [comment]
Your solution worked, thanks for help

**Comment by aluxnimm** [comment]
Dependency and installer update should be fixed in 2.14.1
